requirements/prod.txt

```diff
@@ -356,20 +356,20 @@ typing==3.7.4.1 \
     --hash=sha256:91dfe6f3f706ee8cc32d38edbbf304e9b7583fb37108fef38229617f8b3eba23 \
     --hash=sha256:c8cabb5ab8945cd2f54917be357d134db9cc1eb039e59d1606dc1e60cb1d9d36 \
     --hash=sha256:f38d83c5a7a7086543a0f649564d661859c5146a85775ab90c0d2f93ffaa9714
-pygit2==1.0.2 \
-    --hash=sha256:1176efc3899ee1e0e3fba47a610b6f3e1783385de13d81044da702d5bce15b5b \
-    --hash=sha256:11c5a7b72d8c92d8a7650ae520a8d92c976a680cf47dd61ed92feb0d65575847 \
-    --hash=sha256:11db866707ea174f98d33b9d777409dab1f21ef2332915916e93ee3f1cf4714d \
-    --hash=sha256:25da62e7c4542e8898a6d6ab59775841963ad43a5eb5ec551e534cbb7fe3fc24 \
-    --hash=sha256:3b60c7037454d4d4ac6ddfef5f7afc4c3115143e5c18ae1cb90a8a9e514691fa \
-    --hash=sha256:4b7ffbd09a0e5da3f24c8913498fc628a5f27e171a2e0509890403b6cbe1f333 \
-    --hash=sha256:5ac57ef32d0b7edb57e0dc49a02c52477821539827dcdac415d54c4cf66b1af2 \
-    --hash=sha256:6313c0febdf21173bb6d6088c760b0f9ec3887e8afaa14e53b8003c82cada97c \
-    --hash=sha256:9429a998cfdc76ccea8512277c6581f75a30099fe4dfbe7a5a10c541013180a9 \
-    --hash=sha256:a8cd17ca7ccf3585b031dded74a7dcb00fc3231aa1cc00d98eecbafe20c5378d \
-    --hash=sha256:b7c0abce8cfe4b570e78d16627fe44e76004a96f6cb1c77c8a3ffe8ce37dd2b0 \
-    --hash=sha256:c316248dae9f2a1d79275dc36f7d3200b217931367c6c37eda751d7ea6f441dc \
-    --hash=sha256:d6c1864da421ad41f0548a58b2e2d9d808e55bccde80a8b047aa1027f687213d
+pygit2==1.1.1 \
```

Comment:
oh, I forgot, if pygit2 can't use 1.2 and above then we should pin the dependency to less than 1.2 (or equal to 1.1.1)

Reply:
isn't it already equal to 1.1.1?

Reply:
I wasn't clear, I meant for pyup so it doesn't offer to update it every day.  See the Django requirement at the top of the file for example.

Reply:
gotcha!